README.md

```diff
@@ -8,9 +8,22 @@ The Jib Extension Framework enables anyone to easily extend and tailor the Jib p
 - [1st-party extensions](first-party): extensions developed and maintained by the Jib team
 - [3rd-party extensions](third-party): links to externally developed extensions
 
+
+## Table of Contents
+
+- [Using Jib Plugin Extensions](#using-jib-plugin-extensions)
+   - [Maven](#using-jib-plugin-extensions-maven)
+   - [Gradle](#using-jib-plugin-extensions-gradle)
+- [Writing Your Own Extensions](#writing-your-own-extensions)
+   - [Project Setup](#project-setup)
+   - [Updating Container Build Plan](#updating-container-build-plan)
+   - [Defining Extension-Specific Configuration](#defining-extension-specific-configuration)
+   - [Version Matrix](#version-matrix)
+
+
 ## Using Jib Plugin Extensions
 
-### Maven
+### Maven<a name="using-jib-plugin-extensions-maven"></a>
```

Comment:
Were there too many matching patterns for `#maven`?

Reply:
I think it's easy to accidentally introduce a header with `Maven` like this. So I think this is safer.